src/HttpClientFactory/Http/src/Logging/LoggingHttpMessageHandler.cs

```diff
@@ -19,15 +19,22 @@ namespace Microsoft.Extensions.Http.Logging
     public class LoggingHttpMessageHandler : DelegatingHandler
     {
         private ILogger _logger;
+        private readonly Predicate<string> _isSensitiveHeader;
 
-        public LoggingHttpMessageHandler(ILogger logger)
+        public LoggingHttpMessageHandler(ILogger logger, Predicate<string> isSensitiveHeader)
```

Comment:
This is a breaking API change. This needs to be a separate constructor to avoid the unneccesary break.

Another idea, should this take `HttpClientFactoryOptions` instead? Then we don't have to keep adding constructors if we add more options that impact logging.

Reply:
Please review my changes again, i see only one constructor in the API review code, but you suggest not to introduce breaking changes, could you please suggest what should I do with those constructors?